README.md

```diff
@@ -1,8 +1,10 @@
 ## Your GitHub Learning Lab Repository for Communicating Using Markdown
 
-Welcome to **your** repository for your GitHub Learning Lab course. This repository will be used during the different activities that I will be guiding you through, oh, I haven't introduced myself...
+Welcome to **your** repository for your GitHub Learning Lab course. This repository will be used during the different activities that I will be guiding you through.
 
-I'm the GitHub Learning Lab bot and I'm here to guide you in your journey to learning and mastering the various topics covered in the GitHub Pages course. I will be using Issue and Pull Request comments to communicate with you. In fact I already added an issue for you to checkout.
+Oh! I haven't introduced myself...
```

Comment:
5. I’m jeff cline I’m 45 years old and I’m a Guitar instructor originally when I’m not playing music in one of many projects I’ve started. I like all kinds of music styles.
6.